public/app/common/components/form/style.scss

```diff
@@ -1,4 +1,7 @@
-@import "theme";
+@import "components-theme";
+@import "theme/components/grid";
```

Comment:
They are defined in `project-setup/theme/components`.
Those are default variables overrides for corresponding foundation components.